Recording/Recording.s

```diff
@@ -1,11 +1,12 @@
 # Payload lengths, if any additional data is added, these must be incremented
-.set MESSAGE_DESCRIPTIONS_PAYLOAD_LENGTH, 0x10 # byte count
+.set MESSAGE_DESCRIPTIONS_PAYLOAD_LENGTH, 0x13 # byte count
 .set GAME_INFO_PAYLOAD_LENGTH, 418 # byte count
 .set GAME_INITIAL_RNG_PAYLOAD_LENGTH, 0x8 #byte count
 .set GAME_PRE_FRAME_PAYLOAD_LENGTH, 63 # byte count
 .set GAME_POST_FRAME_PAYLOAD_LENGTH, 0x34 # byte count
+.set GAME_ITEM_INFO_PAYLOAD_LENGTH, 31 # byte count
 .set GAME_END_PAYLOAD_LENGTH, 2 # byte count
-.set FULL_FRAME_DATA_BUF_LENGTH, 8 * (GAME_PRE_FRAME_PAYLOAD_LENGTH + 1) + 8 * (GAME_POST_FRAME_PAYLOAD_LENGTH + 1)
+.set FULL_FRAME_DATA_BUF_LENGTH, 8 * (GAME_PRE_FRAME_PAYLOAD_LENGTH + 1) + 8 * (GAME_POST_FRAME_PAYLOAD_LENGTH + 1) + ((GAME_ITEM_INFO_PAYLOAD_LENGTH + 1) * 15)
```

Comment:
Would be better if the 15 here referenced MAX_ITEMS

Reply:
I tried to do this and realized it was actually pretty hard. You can't really include header files from other header files because the paths are relative to the main code file. Would need to implement absolute paths in `gecko` to support this.

Can you think of another way to do this?